index.js

```diff
@@ -294,6 +294,38 @@ class Statham {
       });
     });
   }
+
+  /**
+   * Search and return keys and values that match given string.
+   *
+   * @param {String} phrase
+   *
+   * @returns {{}}
+   */
+  search(phrase) {
+    let tmp  = {};
+    let data = this.data;
+
+    if (this.isModeNested()) {
+      data = flatten(this.data);
+    }
+
+    Object.getOwnPropertyNames(data).forEach(key => {
+      if (typeof data[key] === 'string' && data[key].indexOf(phrase) !== -1) {
+        tmp[key] = data[key];
+      }
+
+      if (Array.isArray(data[key])) {
+        let arr = JSON.stringify(data[key]);
```

Comment:
Wrong variable name. Because:

1. Always use full names, to keep it readable
2. This is not an array. It's a string representation of what once was an array